src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -301,98 +300,6 @@ public void BindingWorkflow_InstallPackages_Cancellation()
         }
 
         [TestMethod]
```

Comment:
Except that the removal of the product code means the test assemblies are referencing methods which don't exist.